earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TMPFILE=/tmp/FusionTutorial-Full.tar.gz
+SCRIPTDIR=`dirname $0`
```

Comment:
Please, use quotes in your shell script. I know we expect to put this script under a path that doesn't include any shell special characters, but that's not a complete guarantee, since this could be symlinked from any location.